ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionsReader.scala

```diff
@@ -308,6 +308,11 @@ private[dao] final class TransactionsReader(
       .watchTermination()(endSpanOnTermination(span))
   }
 
+  // THIS METHOD WILL BE IMPLEMENTED WITH THE APPEND-ONLY SCHEMA WORK
+  override def getContractStateEvents(startExclusive: (Offset, Long), endInclusive: (Offset, Long))(
```

Comment:
This method will not be implemented for the legacy schema. It's better to just throw here an unsupported exception

Reply:
Done.